### ISSUE TITLE
Device Tracker Docs

### ISSUE THREAD

**Issue by hokfujow** [home-assistant/home-assistant.github.io#23363]
### Feedback

I'm trying to learn how to set up a presence tracker in HA.  Came across the device tracker integration, which talks about different types of devices - eg GPS, router, etc.
It's unclear what devices are compatible , how you would set those up, or if those are part of this integration or you need to integrate something else with this.
In other words, does this integration just provide the logic for looking at multiple trackers and determining the overall presence state? Or does it actually provide the means for tracking specific devices?

### URL

https://www.home-assistant.io/integrations/device_tracker/

### Version

2022.7.3

**Comment by frenck** [comment]
The `device_tracker` is not meant to be used directly, it is used by other integrations to be built on top. This page describes their common capabilities (regardless of which integration 
 provided the entities).

Most integrations in the presence detection category, provide device_tracker entities.

See: <https://www.home-assistant.io/integrations/#presence-detection>

../Frenck

**Comment by hokfujow** [comment]
Thank you so much for the response.  I'm new to HA (but not new to home
automation or tech in general). Have been using X10, X10RF,  Insteon, UPB,
zwave, and zigbee for years.  Up until now I've used Home Control Assistant
(little known but very powerful)  and Hubitat. Getting my feet wet now with
HA and nodered.

 I looked at the list of presence integrations but haven't been able to
determine which (if any) would allow detecting the presence/absence of a
particular bluetooth device by MAC address, when that address is known.
Any suggestions would be appreciated.

Michael
ᐧ
